SwanFileBrowser/src/request.ts

```diff
@@ -0,0 +1,34 @@
+import { URLExt } from '@jupyterlab/coreutils';
+
```

Comment:
What's this file for? Is it a copy from upstream? Any modification you did?

Reply:
This file has the function that allows requesting our backend extensions.
it is not a copy of an upstream function,
I took it from the extension example like this https://github.com/jupyterlab-contrib/search-replace/blob/master/src/handler.ts
I don't remember which one :C